server/api/orders.js

```diff
@@ -0,0 +1,42 @@
+const router = require('express').Router();
+const {User, Order, Product}=require('../db')
+
+//mounted on /api/orders
+router.post('/:userId', async(req,res,next)=>{
+    try{const order=await Order.create()
+        await order.setUser(await User.findByPk(req.params.userId))
+    res.json(order)}
+    catch(err){
+        next(err)
+    }
+})
+router.get('/cart/:userId', async(req,res,next)=>{
+    try {
+        const order=await Order.findAll({include:{
+            model: User,
+            where:{
+                id:req.params.userId,
+                
+            }
+        },where:{
+            status:"cart"
+        }})
+        res.json(order)
+    } catch (err) {
+        next(err)
+    }
+})
+router.put('/:orderId/:productId', async(req,res,next)=>{
```

Comment:
Not a big deal - maybe we can add /:orderId/products/:productId in the route so that it looks like /order/1/products/2